packages/docs/src/lang/en/components/TimePickers.json

```diff
@@ -29,6 +29,10 @@
       "heading": "### AM/PM switch in title",
       "desc": "You can move AM/PM switch to picker's title."
     },
+    "elevation": {
+      "heading": "### Picker's elevation",
+      "desc": "You can set picker's elevation using `elevation` prop or `flat` (the latter sets the elevation to 0)."
```

Comment:
```suggestion
      "desc": "Emphasize the `v-time-picker` component by providing an **elevation** from 0 to 24. Elevation modifies the `box-shadow` css property."
```

Reply:
is different text than for `v-date-picker` intended?

Reply:
yes